mmcv/runner/hooks/logger/text.py

```diff
@@ -138,3 +145,49 @@ def log(self, runner):
 
         self._log_info(log_dict, runner)
         self._dump_log(log_dict, runner)
+
+
+@HOOKS.register_module
+class IterTextLoggerHook(TextLoggerHook):
+
+    def _log_info(self, log_dict, runner):
+        log_str = 'Exp Name: {}\t'.format(runner.meta['exp_name'])
```

Comment:
Add check:
1. if meta is None
2. if 'exp_name' in runner.meta
3. if self.every_n_inner_iters(runner, self.interval_exp_name)